test/Microsoft.AspNetCore.Server.Kestrel.TestCommon/PortManager.cs

```diff
@@ -0,0 +1,20 @@
+using Microsoft.AspNetCore.Server.Testing.Common;
+
+namespace Microsoft.AspNetCore.Server.Kestrel.TestCommon
+{
+    public static class PortManager
+    {
+        public static int _nextPort = 8001;
+        public static object _portLock = new object();
+        
+        public static int GetNextPort()
```

Comment:
Maybe keep the old name GetPort(), to avoid changing all the callers of this API?  I don't think GetNextPort() is a particularly better name.